### ISSUE TITLE
UnsupportedAriaAttribute audit test fails when input has aria-label

### ISSUE THREAD

**Issue by dbatiste** [GoogleChrome/accessibility-developer-tools#178]
The following HTML results in AX_ARIA_10 rule (unsupported aria attribute) failing.  This attribute should be valid.

```javascript
<input aria-label="my test" type="text">
```

This is the first time I've looked at these tools so I am not very familiar with the project, but the `attributeName` check in the following line of code looks suspect to me:

```javascript
if (axs.constants.ARIA_PROPERTIES.hasOwnProperty(lookupName) && !(attributeName in supported)) {
    return true;
}
```

**Comment by RickSBrown** [comment]
Thanks for this. Yes that line is the problem, it should be something like this instead:
```javascript
if (axs.constants.ARIA_PROPERTIES.hasOwnProperty(lookupName) && supported.indexOf(attributeName) < 0) {
    return true;
}
```
 I'll create a PR for it shortly with fix and unit test.

**Comment by dbatiste** [comment]
Thanks Rick!
